### DISCUSSION TITLE
Split NURBS curves/surfaces to Bezier segments/patches

### DISCUSSION THREAD

**Discussion by rendetto** [nortikin/sverchok#4739] [Q&A] [answered]
As mentioned here https://github.com/nortikin/sverchok/pull/4734 

> But, any NURBS curve can be split up into list of Bezier segments.

@portnov  I wonder is there a way to do this for NURBS curves and surfaces in Sverchok?

Thanks

**Comment by portnov** [comment, marked as answer]
For Nurbs curves, there is method: `curve.to_bezier_segments()`. You can use it in scripted node, for example.
For surfaces, it is not implemented yet.

**Reply by rendetto** [reply]
Thanks, I'll try.
